bigtable/google/cloud/bigtable/table.py

```diff
@@ -373,3 +406,50 @@ def _create_row_request(table_name, row_key=None, start_key=None, end_key=None,
         message.rows.row_ranges.add(**range_kwargs)
 
     return message
+
+
+def _mutate_rows_request(table_name, rows):
+    """Creates a request to read rows in a table.
+
+    :type table_name: str
+    :param table_name: The name of the table to read from.
+
+    :type rows: list
+    :param rows: List or other iterable of :class:`.DirectRow` instances.
+    
+    :rtype: :class:`data_messages_v2_pb2.MutateRowsRequest`
+    :returns: The ``MutateRowsRequest`` protobuf corresponding to the inputs.
+    :raises: :class:`TooManyMutationsError <exceptions.TooManyMutationsError>` 
+             if the number of mutations is grater than 100.000
+    """
+    request_pb = data_messages_v2_pb2.MutateRowsRequest(table_name=table_name)
+    mutations_count = 0
+    for row in rows:
+        entry = request_pb.entries.add()
+        entry.row_key = row.row_key
+        for mutation in row._get_mutations(None):
```

Comment:
Done for _check_row. Keep None here, because it is unused in DirectRow but still required bool argument.
_get_mutations is a protected member. Maybe it is better to add a property (getter only) with public access?